src/lib/styles/global/variables.scss

```diff
@@ -8,6 +8,8 @@
   --content-start-height: calc(140px + var(--padding-2x));
 
   --icon-width: 24px;
+  --page-banner-image-width-desktop: 144px;
```

Comment:
The image?

We only thought of two break points, @artkorotkikh-dfinity do you think we need a breakpoint for large screens?

I think that the parent of this component should take care of not increasing the width too much. It's not only the image that will look bad, but the rest as well. And it will depend a lot on the size of the description.

Reply:
Can you provide screenshots?

- I have the feeling that it looks good on laptop but on wide screen, maybe the text will be maybe too spreaded?
- Not sure if 144px is enough for a wide screen and if icon will not look small?

Maybe not at all but just in case rather like that we double check.

Reply:
This is when the banner is 1800px (bigger than what we allow):

![Screenshot 2023-08-22 at 08 16 39](https://github.com/dfinity/gix-components/assets/4550653/feaf4921-255b-492a-9f89-934cf5f27d88)

This is with the maximum width of the parent:

![Screenshot 2023-08-22 at 08 18 57](https://github.com/dfinity/gix-components/assets/4550653/fd816aa3-68ca-494e-99e8-378e51198e78)

Reply:
Sounds good, thanks for double checking.